HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -155,6 +155,25 @@ def customiseFor37046(process):
             delattr(prod, 'PrescaleCSVFile')
     return process
 
+def customizeFor37180(process):
+    for filter in filters_by_type(process, "HLTMuonDimuonL3Filter"):
+        filter.propagatorAlong = cms.ESInputTag("", "hltESPSteppingHelixPropagatorAlong")
+        filter.propagatorAny = cms.ESInputTag("", "SteppingHelixPropagatorAny")
+        filter.propagatorOpposite = cms.ESInputTag("", "hltESPSteppingHelixPropagatorOpposite")
+    for filter in filters_by_type(process, "HLTMuonL3PreFilter"):
+        filter.propagatorAlong = cms.ESInputTag("", "hltESPSteppingHelixPropagatorAlong")
+        filter.propagatorAny = cms.ESInputTag("", "SteppingHelixPropagatorAny")
+        filter.propagatorOpposite = cms.ESInputTag("", "hltESPSteppingHelixPropagatorOpposite")
+    for filter in filters_by_type(process, "HLTMuonTrkFilter"):
+        filter.propagatorAlong = cms.ESInputTag("", "hltESPSteppingHelixPropagatorAlong")
+        filter.propagatorAny = cms.ESInputTag("", "SteppingHelixPropagatorAny")
+        filter.propagatorOpposite = cms.ESInputTag("", "hltESPSteppingHelixPropagatorOpposite")
+    for filter in filters_by_type(process, "HLTMuonTrkL1TFilter"):
+        filter.propagatorAlong = cms.ESInputTag("", "hltESPSteppingHelixPropagatorAlong")
+        filter.propagatorAny = cms.ESInputTag("", "SteppingHelixPropagatorAny")
+        filter.propagatorOpposite = cms.ESInputTag("", "hltESPSteppingHelixPropagatorOpposite")
+
+    return process
```

Comment:
`customizeFor37180` also needs to be called inside `customizeHLTforCMSSW` (the latter contains examples).

Reply:
Done.